execution/src/sce_ledger.rs

```diff
@@ -0,0 +1,332 @@
+use std::sync::{Arc, Mutex};
+
+use crate::ExecutionError;
+use crypto::hash::Hash;
+use models::ModelsError;
+use models::{address::AddressHashMap, hhasher::HHashMap, Address, Amount, AMOUNT_ZERO};
+use wasmer::Module;
+
+/// an entry in the SCE ledger
+#[derive(Debug, Clone, Default)]
+pub struct SCELedgerEntry {
+    pub balance: Amount,
+    pub opt_module: Option<Module>,
```

Comment:
I don't think you need to include `opt_` in the name, since the type makes it clear it's optional.

Reply:
I mostly do that kind of naming to avoid things like `if Some(module) = module`. Similar to what we had elsewhere, like `block.block` etc...